_config.yml

```diff
@@ -1010,4 +1010,11 @@ podcast:
         - title: "Eve Essex on Spotify"
           url: "https://open.spotify.com/artist/1g80vffuPrdapR6S4WyxN3"
         - title: "Eve Essex on Apple Music"
-          url: "https://music.apple.com/gh/artist/eve-essex/id1167165584"
+          url: "https://music.apple.com/gh/artist/eve-essex/id1167165584"    
```

Comment:
Wrong URL?

Reply:
I got confused by this too -- it looks like funny output of the diff, maybe due to line-endings or something.

Reply:
yeah just diff'd it and looks like there used to be some whitespace after the URL: 
![Screen Shot 2021-03-04 at 3 32 23 PM](https://user-images.githubusercontent.com/5361806/110026637-e0346680-7cfe-11eb-9a54-00b6dd2b18c2.png)